vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -265,101 +261,147 @@ external:
+
+  ```yaml title="vcluster.yaml configuration"
+  controlPlane:
+    advanced:
+      defaultImageRegistry: ecr.io/myteam # Replace with your private image registry
+      serviceAccount:
+        imagePullSecrets: 
+          - name: registry-credentials-secret-name # Replace with the name of the registry credentials secret
+      workloadServiceAccount: 
+        imagePullSecrets:
+          - name: registry-credentials-secret-name # Replace with the name of the registry credentials secret
+    
+  external:
+    platform:
+      apiKey:
+        secretName: vcluster-platform-license-key
+  ```
 
   </Step>
+  <Step>
+  Pull the vCluster Helm chart from the private registry.
+  
+  ```bash title="Pull the Helm chart"
+  export VCLUSTER_VERSION=0.25.0 # Replace with the chart version that was pushed into your private registry
+  export REGISTRY=myecr.io/team # Replace with the private registry
+  
+  ## Log in to the Helm registry if you haven't
+  helm registry login ${REGISTRY} \
+    --username myuser \
+    --password mypassword
+  
+  helm pull oci://${REGISTRY}/vcluster:v${VCLUSTER_VERSION}
+  ```
+  </Step>
+  <Step>
 
-<Step>
-Install vCluster Pro
+  Install vCluster into the namespace where you added the secrets.
 
-Use Helm to install the vCluster Pro instance into the `namespace` where you installed the license `secret`:
+  ```bash title="Install vCluster with Helm"
+  export VCLUSTER_NAME="my-vcluster" # Replace with what you want to name your vCluster
+  
+  helm upgrade --install "${VCLUSTER_NAME}" oci://${REGISTRY}/vcluster:v${VCLUSTER_VERSION} \
+    --version ${VCLUSTER_VERSION} \
+    --values vcluster.yaml \
+    --namespace ${VCLUSTER_NAMESPACE} \
+  ```
 
-```bash title="Install vCluster Pro with Helm"
-export VCLUSTER_PRO_NAME="vcluster-pro"
-export VCLUSTER_PRO_NAMESPACE="vcluster-ns"
-helm upgrade --install "${VCLUSTER_PRO_NAME}" vcluster-${VERSION}.tgz \
-  --version ${VERSION} \
-  --values vcluster.yaml \
-  --namespace ${VCLUSTER_PRO_NAMESPACE} \
-```
+  </Step>
+</Flow>
 
-</Step>
+## Air-gapped vCluster with FIPS images {#air-gapped-vCluster-FIPS-images}
 
-</Flow>
-## Air-gapped vCluster Pro with FIPS images
+To run vCluster in a FIPS-compliant environment, you must push FIPS-compliant images to your private registry.
 
-To run vCluster in a FIPS compliant environment, the `vcluster.yaml` needs to be configured to use the Loft GitHub Container Registry with the FIPS compliant images for the syncer (vCluster control plane `StatefulSet`) and the Kubernetes distro images and configured to use the vCluster Pro embedded CoreDNS.
+Push the following images, using the appropriate Kubernetes version as the tag.  
+Tags are listed in the `images.txt` and `images-optional.txt` files.
 
-:::info
-Refer to the [FIPS configuration guide](/vcluster/deploy/security/fips) for more details.
+- `ghcr.io/loft-sh/kubernetes-fips`
+- `ghcr.io/loft-sh/etcd-fips`
+- `ghcr.io/loft-sh/kine-fips`
+- `ghcr.io/loft-sh/helm-fips`
+
```

Comment:
I don't think you need these 3 as they should be embedded already in the `vcluster-pro-fips` image.